packages/server/src/schema/uuid/user/resolvers.ts

```diff
@@ -311,6 +311,12 @@ export const resolvers: LegacyQueries<
       return { success }
     },
 
+    async deleteOwnAccount(_parent, {}, { dataSources, userId }) {
```

Comment:
Here yarn format complains because of the {}, but I don't know how to fix it.

Reply:
```suggestion
    async deleteOwnAccount(_parent, _context, { dataSources, userId }) {
```
or another fitting name.